### PR TITLE
Switch To Class Methods

### PR DESCRIPTION
**Pull request by mgunduz1**
We took our methods inside of the book class. All requirements are met for this milestone.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!

## Required Changes ♻️
_Please check the comments under the review._
- Kindly add a descriptive readme following this [template](https://github.com/microverseinc/readme-template). It keeps the project professional and makes it easy to understand what the project is about.
- Please use the exact eslintrc.json rules stated [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json).


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- Linters
+
+
+## Clone This Project
+```
+- $ git clone `$ git clone -b project1 git@github.com:uwadonat/Awesome_Books.git`
+- $ cd Awesome_Books
+```
+
+
```
Comment (abruzy) [inline README.md]: Good job 👍🏽 
Kindly add a link to the live demo of the app to your readme file

**Review by abruzy — CHANGES_REQUESTED** [review]
```markdown
Hi @mgunduz1 and @uwadonat, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (uwadonat) [reply README.md]: Thanks for your review
A live demo was added to our readme file

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._
- [ ] The live demo doesn't reflect the changes in this branch. Kindly update it to reflect this branch.

Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
 ## Built With
 - HTML5
 - Javascript
 - Linters
 
 
+## Live Demo
+
+[Live Demo Link](https://uwadonat.github.io/Awesome_Books/)
```
Comment (mikenath223) [inline README.md:14]: - [ ] Kindly update the live demo link to reflect the changes in this branch.

Comment (uwadonat) [reply README.md:14]: the live demo is now reflecting the changes.

**Review by hemantso — APPROVED** [review]
Hi @mgunduz1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text


Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**